contrib/pipelines/secadvisor/README.md

```diff
@@ -2,33 +2,223 @@
 
 Implements the IBM Security Advisor pipeline
 
+The security advisor filters the flow data obtained from skydive, performs a data transformation, and saves the information to an object store in JSON format.
+This data may then be used to perform various kinds of analyses for security, accounting, or other purposes.
+We use the term "pipeline" to describe the sequence of operations performed on the data: classify, filter, encode, store.
+In the future, we plan to have numerous pipelines performing different kinds of operations.
 
-## Setup Skydive
+## Deploy Skydive
 
-First build and run skydive:
+Follow instructions on http://skydive.network/documentation/build to install prerequisites and prepare your machine to build the skydive code.
+Build and run skydive:
+
+### All-in-one version
 
 ```
 make
 cp etc/skydive.yml.default skydive.yml
-sudo $(which skydive)) allineone -c skydive.yml
+// adjust settings in skydive.yml
+sudo $(which skydive) allineone -c skydive.yml
+```
+
+### Multi-node Deployment
+Alternatively, run a skydive agent on each of your hosts and a skydive analyzer on one host.
+
+On each machine:
+
+```
+sudo skydive agent -c skydive.yml
+```
+
+Be sure to set the field "analyzers" in the skydive.yml to point to the analyzer.
+
+On one machine:
+
+```
+sudo skydive analyzer -c skydive.yml
+```
+
+
+## Create Security Advisor (Pipeline) Configuration File
+
+Start from the secadvisor.yml.default in the secadvisor directory.
+
+
+```
+cp secadvisor.yml.default secadvisor.yml
+```
+
+The secadvisor.yml is used as a parameter when running secadvisor from the command line.
+
+The default secadvisor.yml has the following fields:
```

Comment:
it don't like the idea of having the entire configuration file embeded - this means that each time we change the configuration file - we need to change this README - a better option (also used for the skydive.yml) is that you add comments within the default configuration file.